distributions/aws/test/e2e/resources/cloudformation-templates/rds-s3.yaml

```diff
@@ -21,83 +23,71 @@ Parameters:
     Default: kubeflow
     Description: Database name for Kubeflow
     Type: String
-    MinLength: '1'
-    MaxLength: '64'
+    MinLength: "1"
+    MaxLength: "64"
     AllowedPattern: "[a-zA-Z][a-zA-Z0-9]*"
     ConstraintDescription: must begin with a letter and contain only alphanumeric characters.
   DBUsername:
-    NoEcho: 'true'
+    NoEcho: "true"
     Description: The database admin account username
     Type: String
-    MinLength: '1'
-    MaxLength: '16'
+    MinLength: "1"
+    MaxLength: "16"
     AllowedPattern: "[a-zA-Z][a-zA-Z0-9]*"
     ConstraintDescription: must begin with a letter and contain only alphanumeric characters.
   DBPassword:
-    NoEcho: 'true'
+    NoEcho: "true"
     Description: The database admin account password
     Type: String
-    MinLength: '8'
-    MaxLength: '41'
+    MinLength: "8"
+    MaxLength: "41"
     AllowedPattern: ".*"
     ConstraintDescription: must contain only alphanumeric characters.
   DBClass:
     Default: db.m5.large
     Description: Database instance class
     Type: String
     AllowedValues:
-    - db.m5.large
-    - db.m5.xlarge
-    - db.m5.2xlarge
-    - db.m5.4xlarge
-    - db.m5.12xlarge
+      - db.m5.large
+      - db.m5.xlarge
+      - db.m5.2xlarge
+      - db.m5.4xlarge
+      - db.m5.12xlarge
     ConstraintDescription: must select a valid database instance type.
   DBAllocatedStorage:
-    Default: '20'
+    Default: "20"
     Description: The size of the database (Gb)
     Type: Number
-    MinValue: '20'
-    MaxValue: '65536'
+    MinValue: "20"
+    MaxValue: "65536"
     ConstraintDescription: must be between 20 and 65536Gb.
   MultiAZ:
     Description: Multi-AZ master database
     Type: String
-    Default: 'false'
-    AllowedValues: ['true', 'false']
+    Default: "false"
+    AllowedValues: ["true", "false"]
     ConstraintDescription: must be true or false.
-  # RDSSecretString:
-  #   Description: Secret string for RDS secrets
-  #   Type: String
+  # See MyRDSSecret and MyS3Secret below
```

Comment:
These were removed?

Reply:
Added them in a previous commit, keeping for reference